tests/jballerina-unit-test/src/test/resources/test-src/expressions/ternary/ternary-expr.bal

```diff
@@ -334,6 +334,22 @@ function testTernaryWithLangValueMethodCalls() {
     assertEquals("a", b48);
 }
 
+function testTernaryWithQuery() {
```

Comment:
Shall we add a few more test cases,
1. Get `i` from function param
2. Get `i` from another module
3. Add module-level ternary expression

Reply:
I am not sure if we can add a test for the third case since type narrowing is not supported at the module level.